<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AlphaFold</string>
  </property>
  <widget class="QDateEdit" name="date">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>130</y>
     <width>110</width>
     <height>30</height>
    </rect>
   </property>
   <property name="locale">
    <locale language="German" country="Germany"/>
   </property>
   <property name="inputMethodHints">
    <set>Qt::ImhDate</set>
   </property>
   <property name="displayFormat">
    <string>dd.MM.yyyy</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="path_in">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>17</x>
     <y>50</y>
     <width>800</width>
     <height>30</height>
    </rect>
   </property>
   <property name="acceptDrops">
    <bool>true</bool>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_output">
   <property name="geometry">
    <rect>
     <x>23</x>
     <y>180</y>
     <width>75</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Output path:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="path_out">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>210</y>
     <width>800</width>
     <height>30</height>
    </rect>
   </property>
   <property name="acceptDrops">
    <bool>true</bool>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="browse_out">
   <property name="geometry">
    <rect>
     <x>830</x>
     <y>210</y>
     <width>50</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QPushButton" name="browse_in">
   <property name="geometry">
    <rect>
     <x>830</x>
     <y>50</y>
     <width>50</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="check_useDate">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>280</width>
     <height>20</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="text">
    <string>Restrict AlphaFold's DB's to a speciffic date</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radio_monomer">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>727</x>
     <y>100</y>
     <width>90</width>
     <height>30</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="text">
    <string>Monomer</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="button_abort">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>550</y>
     <width>100</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>Abort prediction</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_start">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>550</y>
     <width>100</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>Run prediction</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_quit">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>550</y>
     <width>100</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>Quit</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_mode">
   <property name="geometry">
    <rect>
     <x>647</x>
     <y>120</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Model:</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radio_multimer">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>727</x>
     <y>130</y>
     <width>90</width>
     <height>30</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="text">
    <string>Multimer</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_input">
   <property name="geometry">
    <rect>
     <x>23</x>
     <y>20</y>
     <width>130</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Path to .fasta file:</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="text_log">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>260</y>
     <width>860</width>
     <height>280</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
